### PR TITLE
first commit in new branch

### PR DESCRIPTION
**Pull request by pallav-jain-au3**

### CODE REVIEW

File: controller2.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  displayScore(getScoreForP1(), getScoreForP2())
+  startGame();
+}
+
+function startGame() {
+  // attach events celltapped
+  let squares = document.querySelectorAll('.square');
+  squares.forEach(sqaure => square.addEventlistner('click', cellTapped));
+
+  }
```
Comment (sharadJay) [inline controller2.js]: syntactically incorrect

File: controller2.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+function initializeGame(){
+  initializeModel();
+  displayScore(getScoreForP1(), getScoreForP2())
+  startGame();
+}
+
+function startGame() {
+  // attach events celltapped
+  let squares = document.querySelectorAll('.square');
```
Comment (sharadJay) [inline controller2.js]: move this logic into function

File: controller2.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  squares.forEach(sqaure => square.addEventlistner('click', cellTapped));
+
+  }
+}
+
+
+func cellTapped(event) {
+  let gridNumber = event.target.id;
```
Comment (sharadJay) [inline controller2.js]: add a function that passes which move has been made.

File: controller.js [hunk 4]
```diff
@@ -1,23 +1,84 @@
...
+  displayWinningMoves(getWinningArray());
+  displayGameResult(gameStatus);
+  makeScreenUnTappable();
+  updatePlayerScore(gameStatus);
+  displayScores(getScoreForPlayerP1(), getScoreForPlayerP2());
+}
+
+function displayWinningMoves(winningArray) {
+  for (let gridNumber of winningArray) {
+    console.log(gridNumber);
```
Comment (pallav-jain-au3) [inline controller.js]: remove console.log

File: controller.js [hunk 5]
```diff
@@ -1,23 +1,84 @@
...
+function endGame(gameStatus) {
+  displayWinningMoves(getWinningArray());
+  displayGameResult(gameStatus);
+  makeScreenUnTappable();
+  updatePlayerScore(gameStatus);
+  displayScores(getScoreForPlayerP1(), getScoreForPlayerP2());
+}
+
+function displayWinningMoves(winningArray) {
+  for (let gridNumber of winningArray) {
```
Comment (pallav-jain-au3) [inline controller.js]: use foreach

**Pull request merged**